Package.swift

```diff
@@ -248,15 +248,15 @@ packageTargets.append(contentsOf: [
         cSettings: cSettings
     ),
     .target(
-        name: "AwsChecksums",
+        name: "AwsCChecksums",
```

Comment:
Debatable naming: Should we stay consistent with C module naming or with how C modules are named in Swift?

Reply:
This looks good. Name it like the others